pipeline/run_terraform.sh

```diff
@@ -1,23 +1,28 @@
 set -xe
 
-terraform -v
+TERRAFORM=terraform-0.12.29
```

Comment:
Doing it this way will let you specify TERRAFORM in the env.

```suggestion
TERRAFORM="${TERRAFORM:-terraform-0.12.29}"
```

Reply:
@danielhoherd Does this allow us to set that as a pipeline env var via the circle app? Does this default to `terraform-0.12.29`?

Reply:
Yes and yes